backend/static/js/qleverUI.js

```diff
@@ -392,12 +393,61 @@ async function processQuery(sendLimit=0, element=$("#exebtn")) {
     params["cmd"] = "clear-cache";
     nothingToShow = true;
   }
+  const queryId = crypto.randomUUID();
```

Comment:
Integrate a fallback for local (this includes VPNs) testing that also works without HTTPS.